moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp

```diff
@@ -99,13 +99,14 @@ bool PointCloudOctomapUpdater::initialize(const rclcpp::Node::SharedPtr& node)
 
 void PointCloudOctomapUpdater::start()
 {
+  rclcpp::QoS qos(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_sensor_data));
   if (!filtered_cloud_topic_.empty())
-    filtered_cloud_publisher_ = node_->create_publisher<sensor_msgs::msg::PointCloud2>(filtered_cloud_topic_, 10);
-
+    filtered_cloud_publisher_ = node_->create_publisher<sensor_msgs::msg::PointCloud2>(filtered_cloud_topic_, qos);
```

Comment:
hmm.. what effect did the `10` have before this change?

Reply:
This is speculation to some extent, but I believe it will end up calling the `rclcpp::QoS(size_t history_depth)` constructor.

Having said that, I suggest the code is written like this:
`filtered_cloud_publisher_ = node_->create_publisher<sensor_msgs::msg::PointCloud2>(filtered_cloud_topic_, rclcpp::SensorDataQoS()); `